AMW_business/src/main/java/ch/puzzle/itc/mobiliar/business/generator/control/LockingService.java

```diff
@@ -58,15 +58,17 @@
 	 */
 	public DeploymentEntity lockDeployment(Integer id) {
 		int count = 0;
+		int backoff = 300;
 		
-		log.info("Locking deployment " + id);
+		log.fine("Locking deployment " + id);
 		for(; count <= lockRetries; count++) {
 			try {
 				return entityManager.find(DeploymentEntity.class, id, LockModeType.PESSIMISTIC_FORCE_INCREMENT);
 			} catch (LockTimeoutException e) {
-				log.info("Retrying to lock in 200 ms");
+				backoff = getBackoff(backoff);
+				log.info("Retring to lock deployment " + id + " in " + backoff + " ms");
 				try {
-					Thread.sleep(200);
+					Thread.sleep(backoff);
```

Comment:
we should not use Thread.sleep in a Java EE Context, might look for a better solution here